### DISCUSSION TITLE
UWB values

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1290] [Q&A]
Hi, I know the VM client have the plotter for LPS (LoPoTab) to know the UWB ranging distance ?Ｉ want to know how can I put the plot to the value and store like csv file or other file to know the each frame distance. Thanks. 
update...

I use the SD deck but I followed SD card deck usage create a .txt file and store in SD card and then flight the Crazyflie. The SD card will appear the log file but doesn't any data. Thanks.

**Comment by ToveRumar** [comment]
Hi!
Thank you for your question. 

Have you tried the workaround mentioned in this discussion:
https://github.com/orgs/bitcraze/discussions/1234 

This looks like it might be the same bug as mentioned here. It is not yet fixed.
